### ISSUE TITLE
Confusion between person/1166 and person/2073

### ISSUE THREAD

**Issue by nathangibson** [srophe/srophe-app-data#321]
"Bar Shewya or Bar Sabia" http://wwwb.library.vanderbilt.edu/person/1166 and Shabbai http://wwwb.library.vanderbilt.edu/person/2073 are two different people but seem to be somewhat confused. Here is what I can tell:

**person/1166 (Fiey no. 82)**
1. The abstract is correct, except for the names in it. 
2. The versions of the names with "or"/"ou" should be split into two names and one should be chosen as the headword, don't you think? Also "BAR ŜEWYA" should be corrected to BAR ŠEWYA to match Fiey.
3. Dates and attestation are correct.
4. Note with Fiey bibliographic citations is for person/2073 (presumably this can just be deleted).

**person/2073 (Fiey no. 81)**
1. Abstract agrees with Fiey.
2. Should we add the names BAR ŠEWYA (Fiey), "BARŠABBĀ" (Sims-Williams, http://www.iranicaonline.org/articles/barsabba-legendary-bishop-of-marv-and-founder-of-the-christian-church-in-eastern-iran), "Bar Shabba", "Mar Shabba", "ܒܪܫܒܐ" (Chabot, 43, https://archive.org/stream/ChabotSynodiconOrientale/chabot synodicon orientale#page/n61), "القديس برشبا" and "Baršaba" (Scher, Seert, https://archive.org/stream/ScherChronicleOfSeertcombined1/scher chronicle of seert (combined) (1)#page/n138) ? At the Salzburg conference, people were referring to him as "Bar Shabba" -- I wonder if this is more standard and we should switch the headword?
3. I can't tell from work/1525 whether the "attestation" is correct, can you?
4. The bibliography does not have the items listed in Fiey's Note.
5. Date of office c. 370-not before 424, if I'm understanding Fiey's French correctly.

**Comment by nathangibson** [comment]
Also, http://www.iranicaonline.org/articles/barsabba-legendary-bishop-of-marv-and-founder-of-the-christian-church-in-eastern-iran identifies some additional texts for person/2073, but perhaps this is too much for now!

**Comment by davidamichelson** [comment]
Yes, no need to add new stuff!!! We are trying to get this out the door. :)

**Comment by jsaintlauren** [comment]
Thanks, Nathan!  

I will comment on your comments one by one. 

person/1166 (Fiey no. 82)
1. The versions of the names with "or"/"ou" should be split into two names and one should be chosen as the headword, don't you think? YES -  Let's just call Bar Sabia. 
 
This work: http://wwwb.library.vanderbilt.edu/work/1525 should be linked to him, person 1166.  
Also "BAR ŜEWYA" should be corrected to BAR ŠEWYA to match Fiey. - YES 

2. Note with Fiey bibliographic citations is for person/2073 (presumably this can just be deleted). _ YES please delete note. 

person/2073 (Fiey no. 81)

2. Should we add the names BAR ŠEWYA (Fiey), "BARŠABBĀ" (Sims-Williams, http://www.iranicaonline.org/articles/barsabba-legendary-bishop-of-marv-and-founder-of-the-christian-church-in-eastern-iran), "Bar Shabba", "Mar Shabba", "ܒܪܫܒܐ" (Chabot, 43, https://archive.org/stream/ChabotSynodiconOrientale/chabot synodicon orientale#page/n61), "القديس برشبا" and "Baršaba" (Scher, Seert, https://archive.org/stream/ScherChronicleOfSeertcombined1/scher chronicle of seert (combined) (1)#page/n138) ? At the Salzburg conference, people were referring to him as "Bar Shabba" -- I wonder if this is more standard and we should switch the headword?
Yes to all of this.  Fine to refer to him as Bar Shabba 
Please add in the bibliography from FIey to his record.

**Comment by jsaintlauren** [comment]
Let's give Bar Shabba (person 2073) a floruit of 424 and leave it at that.

**Comment by wlpotter** [comment]
Left to do here, from what I can tell based on the above comments

- [ ] The headword for https://syriaca.org/person/1166 is Bar Shebya, should this be Bar Sabbia?
- [ ] The work https://syriaca.org/work/1525 should commemorate https://syriaca.org/person/1166. Currently it's on https://syriaca.org/person/2073, should that be removed?
- [ ] BAR ŜEWYA should be corrected to BAR ŠEWYA in https://syriaca.org/person/1166
- [ ] The list of name variants should be added to https://syriaca.org/person/2073
> Should we add the names BAR ŠEWYA (Fiey), "BARŠABBĀ" (Sims-Williams, http://www.iranicaonline.org/articles/barsabba-legendary-bishop-of-marv-and-founder-of-the-christian-church-in-eastern-iran), "Bar Shabba", "Mar Shabba", "ܒܪܫܒܐ" (Chabot, 43, https://archive.org/stream/ChabotSynodiconOrientale/chabot synodicon orientale#page/n61), "القديس برشبا" and "Baršaba" (Scher, Seert, https://archive.org/stream/ScherChronicleOfSeertcombined1/scher chronicle of seert (combined) (1)#page/n138) 
- [ ] Bar Shabba as the headword for https://syriaca.org/person/2073
- [ ] Fiey bibliography for https://syriaca.org/person/2073 should be added (entry 81)
- [ ] circle back to the https://www.iranicaonline.org/articles/barsabba-legendary-bishop-of-marv-and-founder-of-the-christian-church-in-eastern-iran entry for https://syriaca.org/person/2073 and potentially add more commemorations and bibliography
